targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L443xC/device/TOOLCHAIN_ARM_STD/stm32l443xx.sct

```diff
@@ -29,36 +29,82 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 #if !defined(MBED_APP_START)
-  #define MBED_APP_START 0x08000000
+  #define MBED_APP_START            0x08000000
 #endif
 
+; 256KB FLASH (0x40000)
 #if !defined(MBED_APP_SIZE)
-  #define MBED_APP_SIZE 0x40000
+  #define MBED_APP_SIZE             0x40000
+#endif
+
+; 48KB SRAM (0xC000) + 16KB SRAM (0x4000)
+#if !defined(MBED_RAM_START)
+  #define MBED_RAM_START            0x20000000
+#endif
+
+; RW data 48k L4-SRAM1
+#if !defined(MBED_RAM_SIZE)
+  #define MBED_RAM_SIZE             0xC000
+#endif
+
+#if !defined(MBED_RAM2_START)
```

Comment:
This macro comes from CMSIS pack ?
So is it mandatory to always define a default value ?

Reply:
Yes because the macro is yet not available for all targets.